coordinator/src/services/translation/cancellation/cancellation-response.ts

```diff
@@ -48,43 +48,43 @@ function createBundleEntries(cancellationResponse: CancellationResponse) {
   const hl7AuthorAgentPerson = cancellationResponse.author.AgentPerson
 
   const {
-    fhirPractitioner: fhirResponsiblePartyPractitioner,
-    fhirLocations: fhirResponsiblePartyLocations,
-    fhirHealthcareService: fhirResponsiblePartyHealthcareService,
-    fhirPractitionerRole: fhirResponsiblePartyPractitionerRole
+    fhirPractitioner: fhirCancelRequesterPractitioner,
+    fhirLocations: fhirCancelRequesterLocations,
+    fhirHealthcareService: fhirCancelRequesterHealthcareService,
+    fhirPractitionerRole: fhirCancelRequesterPractitionerRole
   } = convertAgentPerson(hl7ResponsiblePartyAgentPerson)
```

Comment:
This is the wrong way round. ResponsibleParty in the HL7 V3 message is the author of the original prescription. Author is the author of the cancellation message.